esmvaltool/recipes/recipe_climwip_brunner20esd.yml

```diff
@@ -0,0 +1,244 @@
+      dimension: time
+      method: linear
+    climate_statistics:
+      operator: std_dev
+
+  annual_trend:
+    <<: *general
+    annual_statistics:
+      operator: mean
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: "K year-1"
+
+  global_mean:
+    <<: *general
+    climate_statistics:
+      operator: mean
+    area_statistics:
+      operator: mean
+
+  temperature_anomalies:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference: &ref_period
+        start_year: 1995
+        start_month: 1
+        start_day: 1
+        end_year: 2014
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  # if local anomalies are needed as predictors they need to be calculated first
+  calculate_local_anomalies:
+    variables:
+      tas_CLIM: &performance_settings
+        short_name: tas
+        start_year: 1980
+        end_year: 2014
+        mip: Amon
+        preprocessor: climatological_mean
+        additional_datasets: *obs_data
+      tas_GLOBAL:
+        <<: *performance_settings
+        preprocessor: global_mean
+      psl_CLIM:
+        <<: *performance_settings
+        short_name: psl
+        preprocessor: climatological_mean
+      psl_GLOBAL:
+        <<: *performance_settings
+        short_name: psl
+        preprocessor: global_mean
+    scripts:
+      local_anomalies:
+        script: weighting/calculate_difference_variable_group.py
+        obs_data: native6
+
+
+  calculate_weights_climwip:
+    variables:
+      tas_CLIM_i: &independence_settings
+        short_name: tas
+        start_year: 1980
+        end_year: 2014
+        mip: Amon
+        preprocessor: climatological_mean
+        additional_datasets: *obs_data
+      psl_CLIM_i:
+        <<: *independence_settings
+        short_name: psl
+      tas_STD:
+        <<: *performance_settings
+        preprocessor: detrended_std
+      psl_STD:
+        <<: *performance_settings
+        short_name: psl
+        preprocessor: detrended_std
+      tas_TREND:
+        <<: *performance_settings
+        preprocessor: annual_trend
+      tas_CLIM_reference: &target_settings
+        short_name: tas
+        start_year: 1995
+        end_year: 2014
+        mip: Amon
+        preprocessor: climatological_mean
+      tas_CLIM_future:
+        <<: *target_settings
+        start_year: 2081
+        end_year: 2100
+
+    scripts:
+      climwip:
+        script: weighting/climwip/main.py
+        obs_data: native6
+        ancestors: [calculate_local_anomalies/local_anomalies,
+                    tas_CLIM_i, psl_CLIM_i, tas_STD, psl_STD, tas_TREND,
+                    tas_CLIM_reference, tas_CLIM_future]
+        combine_ensemble_members: true
+        performance_sigma: 0.43
+        # calibrate_performance_sigma:
+        #   target_ref: tas_CLIM_reference
+        #   target: tas_CLIM_future
```

Comment:
```suggestion
```

Reply:
I prefer to leave this in to make, to make it east to re-calculate the sigma. I will add a comment and line in the documentation